### PR TITLE
Show dock icon of the app.

### LINKED ISSUE #114: Mac Installer UX Suggestion
1. Add Icon on dock as well, versus just having it on the status bar 
screen shot 2015-07-23 at 1 31 31 pm-01

2. Dock Icon opens to another window which there are clear instructions to Turn KA lite on/off and to open KA lite in browser

### PR DESCRIPTION
**Pull request by amodia**
@cpauya & @aronasorman  this fixes #114

I enabled the property to show icon on the dock. I've attached a screenshot.

screen shot 2015-10-19 at 5 42 12 pm

### CODE REVIEW

**Comment by cpauya** [comment]
@amodia Is this only displaying the icon?  What happens if I click it during:

1. Preferences window is open?
2. Preferences window is not open?

Use `Finder` as an example on the default behaviour of dock icons.

**Comment by cpauya** [comment]
@amodia As per our demo yesterday, pls fix the issue with multiple `Quit` menu option showing when the user right-click on the dock menu if it's an easy fix.  Otherwise, you can comment that and we can merge this in.

Can you attach a screenshot of it here?

**Comment by amodia** [comment]
@cpauya I already removed the menu bar at dock icon. I think you can merge this PR. I've attached a sample screenshot.

screen shot 2015-10-27 at 1 02 27 pm

File: osx/KA-Lite-Monitor/KA-Lite-Monitor/AppDelegate.m [hunk 1]
```diff
@@ -55,6 +55,17 @@ - (void) extractAssessment{
...
 
+//REF http://objcolumnist.com/2009/08/09/reopening-an-applications-main-window-by-clicking-the-dock-icon/
+- (BOOL)applicationShouldHandleReopen:(NSApplication *)theApplication hasVisibleWindows:(BOOL)flag {
+    if(flag==NO) {
+        [window makeKeyAndOrderFront:self];
+    }
+    return YES;	
+}
+
+//- (NSMenu *)applicationDockMenu:(NSApplication *)sender {
```
Comment (cpauya) [inline osx/KA-Lite-Monitor/KA-Lite-Monitor/AppDelegate.m]: What's this commented code for @amodia ?

**Comment by cpauya** [comment]
Tested, merging.

**Pull request merged**